syslibs/opcua/include/helpers.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * helpers.h

```

Comment:
The `include/` directory of OV libs is added as header search path *publicly* (see [ov_functions.cmake:27](https://github.com/acplt/rte/blob/master/cmake/ov_functions.cmake#L27)), i.e. other libraries depending on the opcua library will also search for header files in opcua's include directory. Thus, `opcua_helpers.h` was actually the better file name, since it will less likely result in naming collisions.

Reply:
So, i guess we should add the library prefix for other headers too?
* nsSwitch.h
* ovSwitch.h
* ovTrafo.h

Reply:
Should we keep the source file names without library prefix?

Reply:
For the source files it should not make any difference. However, it is a nice convention to have the corresponding .h and .c files' names match to make it easier to explore the code.